lib/Service/ConnectivityService.php

```diff
@@ -47,6 +51,38 @@ public function testCapabilities(OutputInterface $output): void {
 		$output->writeln('<info>✓ Detected WOPI server: ' . $this->capabilitiesService->getServerProductName() . ' ' . $this->capabilitiesService->getProductVersion() . '</info>');
 	}
 
+	public function testCallback(OutputInterface $output): void {
+		$url = $this->parser->getUrlSrcValue('Capabilities');
+		if ($url === '') {
+			// Fixme can we skip early if the collabora version does not have the wopiAccessCheck endpoint, maybe it can be exposed in discovery
+			return;
+		}
+
+		$url = str_replace('/hosting/capabilities', '/hosting/wopiAccessCheck', $url);
+
+		$callbackUrl = $this->urlGenerator->getAbsoluteURL('/status.php');
+
+		$client = $this->clientService->newClient();
+		try {
+			$response = $client->post($url, [
+				...RemoteOptionsService::getDefaultOptions(),
+				'body' => json_encode([
+					'callbackUrl' => $callbackUrl,
+				]),
+				'headers' => [
+					'Content-Type' => 'application/json',
+				],
+			]);
+			$result = json_decode($response->getBody(), true, 512, JSON_THROW_ON_ERROR);
+			if ($result['status'] === 'CheckStatus::WopiHostNotAllowed') {
```

Comment:
```suggestion
			if ($result['status'] === 'WopiHostNotAllowed') {
```

I have simplified this to just be `WopiHostNotAllowed` in https://github.com/CollaboraOnline/online/pull/10807

I am taking the time window until mid-january and next COOL stable version to make small API corrections before they are completely set in stone.
You can see the MR related at the end of https://github.com/CollaboraOnline/online/pull/9202